### PR TITLE
Add taxcalc recipe

### PR DESCRIPTION
**Pull request by chusloj**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by chusloj** [comment]
@conda-forge/staged-recipes @conda-forge/help-python Ready for review. Thank you!

**Comment by chusloj** [comment]
@conda-forge/staged-recipes Ready for review!

**Comment by chusloj** [comment]
@MattHJensen Are you OK with being listed as a maintainer here?

**Comment by MattHJensen** [comment]
> @MattHJensen Are you OK with being listed as a maintainer here?

Yes, thanks @chusloj.

**Comment by chusloj** [comment]
@conda-forge-admin Ready for review, thank you!

**Comment by chusloj** [comment]
@conda-forge-admin, please ping team

File: recipes/taxcalc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  host:
+    - python >=3.7
+    - pip
+    - numpy >=1.14
+    - pandas >=1.2.0
+    - bokeh >=1.4.0
+    - requests
+    - numba
+    - paramtools >=0.18.0
+    - aiohttp
```
Comment (chrisburr) [inline recipes/taxcalc/meta.yaml:29]: You probably don't need these in `host`
```suggestion
```

Comment (chusloj) [reply recipes/taxcalc/meta.yaml]: Thanks @chrisburr

**Comment by chusloj** [comment]
@conda-forge-admin, please ping team

**Comment by chusloj** [comment]
@chrisburr Thanks for your help!

**Pull request merged**